### PR TITLE
Workaround for Cygwin and tag correct branch on finish commands

### PR DESCRIPTION
**Pull request by Gawen-pjr**

### CODE REVIEW

File: contrib/gitflow-installer.sh [hunk 1]
```diff
@@ -17,7 +17,7 @@ if [ -z "$REPO_NAME" ] ; then
 fi
 
 if [ -z "$REPO_HOME" ] ; then
-	REPO_HOME="http://github.com/nvie/gitflow.git"
+	REPO_HOME="http://github.com/Gawen-pjr/gitflow.git"
```
Comment (nfalco79) [inline contrib/gitflow-installer.sh:20]: maybe it's unwanted changes.

**Comment by Gawen-pjr** [comment]
Ah sorry for that. It was a patch to test the modifications from my own github repository. It's effectively unwanted changes

**Comment by Shea690901** [comment]
Then «close the pull-request» ;)

**Comment by Gawen-pjr** [comment]
New pull-request with unwanted changes removed  : #6366

**Pull request closed without merging**